File: S03-conceptos-basicos/CL02-CiclosVidaActivity/app/src/main/java/com/udemyandroid/activity/MainActivity.java
package com.udemyandroid.activity;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.i("LOGCAT", "entra en el método onCreate");
    }

    @Override
    protected void onStart() {
        super.onStart();
        Log.i("LOGCAT", "entra en el método onStart");
    }

    @Override
    protected void onResume() {
        super.onResume();
        Log.i("LOGCAT", "entra en el método onResume");
    }

    @Override
    protected void onPause() {
        super.onPause();
        Log.i("LOGCAT", "entra en el método onPause");
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        Log.i("LOGCAT", "entra en el método onDestroy");
    }
}

File: S13-kotlin/CL11-ActivityLifecycle/app/src/main/java/com/androidavanzado/activitylifecycle/MainActivity.kt
package com.androidavanzado.activitylifecycle

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Log.i("MainActivity", "método onCreate")
    }

    override fun onStart() {
        super.onStart()
        Log.i("MainActivity", "método onStart")
    }

    override fun onResume() {
        super.onResume()
        Log.i("MainActivity", "método onResume")
    }

    override fun onPause() {
        super.onPause()
        Log.i("MainActivity", "método onPause")
    }

    override fun onStop() {
        super.onStop()
        Log.i("MainActivity", "método onStop")
    }

    override fun onDestroy() {
        super.onDestroy()
        Log.i("MainActivity", "método onDestroy")
    }
}
